cmd/loom/gateway/mapping_cmds.go

```diff
@@ -374,7 +375,7 @@ func newMapAccountsCommand() *cobra.Command {
 }
 
 const ListContractMappingCmdExample = `
-loom gateway list-contract-mappings
+loom gateway list-contract-mappings <output-format>?
```

Comment:
There should be an optional boolean flag `--raw`, when specified the command should output plain JSON.